app/routers/admin.py

```
@@ -152,60 +185,64 @@ async def dashboard(
+                    "total_alarms": total_alarms,
+                    "total_alarms_sent": total_alarms_sent,
+                    "success_rate": success_rate,
+                    "failed_sends": failed_sends
+                }
             }
-        }
-    )
+        )
+    except Exception as e:
+        logger.exception("Admin dashboard rendering failed")
+        raise HTTPException(status_code=500, detail=f"Admin dashboard failed: {str(e)}") from e
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)